package.json

```
@@ -4,7 +4,7 @@
   "description": "GoFSH is a FHIR Shorthand (FSH) decompiler, able to convert formal FHIR definitions from JSON to FSH.",
   "main": "dist/index.js",
   "scripts": {
-    "build": "rm -rf dist && tsc && cpy src/utils/template.html dist/utils",
+    "build": "rm -rf dist && tsc && copyfiles -u 3 \"src/utils/template.html/*\" dist/utils",
```

Since this is just copying one file, I think this needs to be changed to not use a glob.
```suggestion
    "build": "rm -rf dist && tsc && copyfiles -u 2 \"src/utils/template.html\" dist/utils",
```